docs/development.md

```diff
@@ -180,10 +180,10 @@ skip to {ref}`sec_development_workflow_anothers_commit`.
    a local copy as detailed in {ref}`sec_development_getting_started_environment`.
 
 2. Make sure that your local repository has been configured with an
-   [upstream remote](https://help.github
-   .com/articles/configuring-a-remote-for-a-fork/):
+   [upstream remote](
+   https://help.github.com/articles/configuring-a-remote-for-a-fork/):
    ```bash
-   $ git remote add upstream git@github.com:tskit-dev/tskit.git
```

Comment:
I'm not sure about this change - I tend to use ssh access, but maybe most people don't?

I think the correct http link is ``https://github.com/tskit-dev/tskit.git`` in any case though.

Reply:
For the upstream, most folks wont have authenticated access and so must use the non-authenticated https url.

Reply:
Ah - thanks @grahamgower . OK, we just need to change to ``https://github.com/tskit-dev/tskit.git`` then

Reply:
yeah, exactly! it works with or without the `.git` extension, but I'm happy to add it back in.

Reply:
done!